src/components/utils/onfidoApi.js

```diff
@@ -27,6 +27,18 @@ export const uploadLivePhoto = (data, token, onSuccess, onError) => {
   sendFile(endpoint, data, token, onSuccess, onError)
 }
 
+export const getLivenessChallenges = () => {
+  return Promise.resolve([
```

Comment:
Hard-coded for now

Reply:
I would avoid depending on promises so leave room to remove that dependency in the future and decrease bundle size.